english/candyland/diabetes.md

```diff
@@ -0,0 +1,20 @@
+You light a small fire in the middle of the marshmallow room.
+
+As the fire grows, a sweet aroma fills the room and the ceiling becomes ever so slightly brown just above the fire.
+
+The marshmallow room started melting and you started seeing some colorful lights shining through.
+
+Wait! But this is not any colorful lights you are seeing!  What appears before you is a rainbow colored Gumdrop Mountains!
+
+Surrounding the Gumdrop Mountain are acres of Lollipop Woods where there's a forest of giant lollipops as tall as sequoia.
+
+You wanted to taste all these sweetness around you, but damn you just realize you have diabetes.
+
+
+[Light fire] (light-fire/fire.md)
+
+[Save yourself!](run-away/outside.md)
+
+[Begin to eat!](eating/inside.md)
+
+[Turn on the chocolate sprinkler!](sprinkler/eating-chocolate.md)
```

Comment:
I fixed all the broken links since I forgot to commit the nested files.  I also broke up long sentence.  Hope this works!